src/pages/partner/index.md

```diff
@@ -414,103 +396,34 @@ A typical event will have the following information.
-/JPWYd2pNXvZAgMBAAECggEAVWdk6jP0TeRMT+idqFIJYIDexbaOSdO71wPCPnX30uloWkz3XIwa
-ntq+Qcc98372SqzY/jCkbhnJ/bZt4YUi2y6TJvJ6FZLTZFS39F6Y873XN/S229Dh+YgC+t65RtiX
-vKe4rBjcLubX561zXHUWjDK51LoFu0rXXXXXXXXXX92bKNqmktyNjuvJpmNWHhOBmRwftc1c86QK
-MwsA1mqhEt77cRW8rTSeRB0qMkoHKhjTo7HZfOLSbd/iqGtF6V/Fgzh0MOoPh/VXMjFC79GV74ZI
-WArQ7OiT8BP/GKyjcNg3QOfEL2Ry//JA3fG7iyq1ssxUlcTmh8xSmZIK2iQxgQKBgQDSBxUe8hIU
-VM607sFabQ+pqI/Tcwui0HoaJ0Tq8jpXXXXXXXXXXH9R2Q4z41zvfbkCvesWHolflsl8xizRv9bI
-Z+Ot7bOYsEaTE+ZrVHy9e9GnCXM0eXwqjSMW/DdNqLj3c0NQTR2nIPRnvKxL+HE74Sny1HZa/fgn
-QN9uRpkhtQKBgQC48emjE2Qw3CVqqv/MIX2Qrdf/JjCM2uxPFV+igbMTUd1G3gRp9zjPMZQ6xvBP
-Rvt0Ao/guQ/lmfM6fI83YCz5W/LYmWgz7j+qerxUQeM4G9R/TENFzIZLBZJkdaSkWvUSzmZdX4bT
-D7NGe6NbdC50KrNg476kqJ5rGYCgtoHYFQKBgAOMMRQjvKBxSkiOG9/+JZyVB1DoCZNHeDiG30gk
-rU4p7coJZrnQbz4fr3i652/JLK7aLGVkQKfHsUGVS16Dogoijxr5/+WdXKPRWzwG4lCFDeaHZ1XE
-k49YcPZLble2eOFdJg7uLokLKnWyHyrtWGRNilraseEZlx56nwuKz71dAoGBALNWcZYesaLndqju
-PEjfmOD4kmDqaV3M1CUDts7gMgPj+ExF8cXHYyEXFIPCcTkiIncmHNpu2cls9XybWEz8R+Atl1kN
-uQvSVY48oe6aTXkVE4O1Me63GuoYpxIfOd3eDZOE7aJRhW2Y5eDM92UxTfJ7jaHAdhc5IxT7j81H
-MLTZAoGABBBtyVUqrRXj8t+LhblWvZkQLgr2tk+n7cKI56ztc3R5TAG0rMPDy7XJ9Mb6t0DCHtCA
-FsxGx1+W6u7/PG9Hrp6L8N3ONLeS4//cCIT2Qu+6ZiSaCLtglaPwBeUQclpM0DtQmHbrq53a2CIt
-FPFUIzEwtY8zhZRPnCeBO62uMsA=
------END PRIVATE KEY-----
-```
-
-
-
-## Sample JWT
-A sample JWT is displayed below. 
+A sample access_token response from IMS is displayed below.
 
 ```java
 {
-"CLIENT_SECRET":"s8e-glpSghksnMgQdg8sHYQ18UZUxP8ejOFK",
-"ORG_ID":"<your_org>@ADOBEOrg",
-"API_KEY":"3f5f26c871ac45e985867adcfa84d180",
-"TECHNICAL_ACCOUNT_ID":"<your_account>@techacct.ADOBE.com",
-"TECHNICAL_ACCOUNT_EMAIL":"<your_email>@techacct.ADOBE.com",
-"PUBLIC_KEYS_WITH_EXPIRY":{"84a06e0e6ad1bb822b2b14f6e628d6f46099f06d":"08/27/2022"}
+    "access_token": "eyJhbGciOiJSUzI1NiIsIng1dSI6Imltc19uYTEtc3RnMS1rZXktYXQtMS5jZXIiLCJraWQiOiJpbXNfbmExLXN0ZzEta2V5LWF0LTEiLCJpdHQiOiJhdCJ9.eyJpZCI6IjE2OTU4OTQ3NTYwOTdfYWYyMWQ2OTktNzkwNy00NjNmLThhOTktODk5Njc3OTUxNTE3X2V3MSIsIm9yZyI6IjM1QTk0M0MwNjJEQTY0MTEwQTQ5NDIzN0BBZG9iZU9yZyIsInR5cGUiOiJhY2Nlc3NfdG9rZW4iLCJjbGllbnRfaWQiOiIyYmIyZmViMGZkZWM0ZmRlODk1MTY0NGJhNzcwZTMyOSIsInVzZXJfaWQiOiI3OTlCMUE5MjY1MDJFNEM1MEE0OTQwMDlAdGVjaGFjY3QuYWRvYmUuY29tIiwiYXMiOiJpbXMtbmExLXN0ZzEiLCJhYV9pZCI6Ijc5OUIxQTkyNjUwMkU0QzUwQTQ5NDAwOUB0ZWNoYWNjdC5hZG9iZS5jb20iLCJjdHAiOjMsIm1vaSI6IjcwNTg2OWI3IiwiZXhwaXJlc19pbiI6Ijg2NDAwMDAwIiwiY3JlYXRlZF9hdCI6IjE2OTU4OTQ3NTYwOTciLCJzY29wZSI6ImFkb2JlaW9fYXBpLG9wZW5pZCxyZWFkX2NsaWVudF9zZWNyZXQsQWRvYmVJRCxhZGRpdGlvbmFsX2luZm8ucm9sZXMsbWFuYWdlX2NsaWVudF9zZWNyZXRzLHJlYWRfb3JnYW5pemF0aW9ucyxhZGRpdGlvbmFsX2luZm8ucHJvamVjdGVkUHJvZHVjdENvbnRleHQifQ.g3eBFADKN6q9gxMiiAL53MxM7FQGDO3LSahtpcp2E7irEJ6xWjrhc28rFEFoZZc3rRBDWiUqGj2faVopUZmNZ0HbKl-QcXfM9yYRzsBm9y8Zw91RbvieNU1MqxkYbCS-TzcFBAgGGPcdyyWylT_2WgAIUiXI9K5CyR2tRaScqwFhAS8S371AA7Hq3QMyJUGwZwqxYMbfKy2jh1ZNYyCLkRHXMwl2FUNgjk2pQiOnetbSsI0muxwu_cOoE8i6A9TA8CY4vZ22zSCCaQXXeKG2jGB1qGEz1CaRCEutYPcY-Lxx1YwvN73L7nrLR8_yZ5_2MKK_fm9VjhmViQHgsQ",
```

Comment:
I hope this is altered enough not be an actual token.

Reply:
Yes, I'd removed few characters in between.